packages/env/lib/cli.js

```diff
@@ -46,31 +46,22 @@ module.exports = function cli() {
 	yargs.usage( wpPrimary( '$0 <command>' ) );
 
 	yargs.command(
-		'start [ref]',
```

Comment:
This needs a dev note, right?

Reply:
I noted that this breaks backwards compatibility in `packages/env/CHANGELOG.md` but a agree dev note would be good, too. If nothing else it will help publicise that this tool can be used for core development.

Reply:
:+1: